src/test/run-make/cabi-int-widening/Makefile

```diff
@@ -0,0 +1,10 @@
+-include ../tools.mk
+
+all: $(call NATIVE_STATICLIB,cfoo)
```

Comment:
This might be most easily added actually as part of `src/rt/rust_test_helpers.c` and then a standard `run-pass` test can be added. Helps avoid `run-make` at least!1